### DISCUSSION TITLE
Ability to sort the recipe/device image/hardware list by name or other variables

### DISCUSSION THREAD

**Discussion by Ahreluth** [Genymobile/feedback#67] [Genymotion SaaS] [answered]
This would be a life saver, as my devices are currently numerically numbered, but they come up in a random order. Having the ability to sort this list in some way would really help productivity.

**Comment by thomascarpentier** [comment, marked as answer]
Hi @Ahreluth, 

Thanks for your feedback! This is a feature that we are already thinking of adding to our product. We do not have any ETA for the moment, but we can contact you when it is available if you wish.

**Comment by bhein-sherpa6** [comment]
+1 for this. This would be a big quality of life improvement. Cheers!

**Comment by PATAPOsha** [comment]
+1

**Comment by Saeidto68** [comment]
BugFreeNet24!
